applications/fax/src/fax_worker.erl

```diff
@@ -865,11 +862,11 @@ prepare_contents(<<"application/pdf">>, JobId, RespContent, TmpDir) ->
     end;
 
 prepare_contents(<<"image/", SubType/binary>>, JobId, RespContent, TmpDir) ->
-    InputFile = list_to_binary([TmpDir, JobId, ".", SubType]),
+    InputFile = list_to_binary([TmpDir, JobId, "-input.", SubType]),
```

Comment:
When input is TIFF file, then input and output file must have different filename